spring-integration-core/src/main/java/org/springframework/integration/support/leader/LockRegistryLeaderInitiator.java

```diff
@@ -183,12 +198,22 @@ public void setBusyWaitMillis(long busyWaitMillis) {
 
 	/**
 	 * Sets the {@link LeaderEventPublisher}.
+	 *
 	 * @param leaderEventPublisher the event publisher
 	 */
 	public void setLeaderEventPublisher(LeaderEventPublisher leaderEventPublisher) {
 		this.leaderEventPublisher = leaderEventPublisher;
 	}
 
+	/**
+	 * Sets the {@link ExecutorService}
+	 *
+	 * @param executorService the executor service
```

Comment:
The method JavaDocs should be like: 
- Use an imperative style (i.e. Return and not Returns) for the first sentence.
- No blank lines
- Must be `@since` when it is new public API.

https://github.com/spring-projects/spring-framework/wiki/Code-Style#javadoc-formatting